Formula/sbt.rb

```diff
@@ -6,6 +6,13 @@ class Sbt < Formula
   sha256 "86a6f28e79966074101066ab1aa19267d0c8d44eae21daa93c9c6056ca6f9da4"
   license "Apache-2.0"
 
+  # cannot use `https://www.scala-sbt.org/download.html`,
+  # as it still lists `1.4.5` as the latest release
```

Comment:
```suggestion
```

The formula is using an archive from GitHub, so the `livecheck` block here is aligned with the `stable` source. I don't see any issues that would lead us to need to check the first-party download page, even if it was up to date.

With that in mind, we can simply omit this comment, as it may confuse some people rather than help.

Reply:
Sounds good to me, maybe I should file a ticket to point out the outdated release information on the first party download page.